supervision/detection/tools/inference_slicer.py

```diff
@@ -4,20 +4,28 @@
 import numpy as np
 
 from supervision.detection.core import Detections
-from supervision.detection.utils import move_boxes
+from supervision.detection.utils import move_boxes, move_masks
 from supervision.utils.image import crop_image
 
 
-def move_detections(detections: Detections, offset: np.array) -> Detections:
+def move_detections(
+    detections: Detections, offset: np.ndarray, image_shape: np.ndarray
```

Comment:
Argument `image_shape` is called `resolution_wh` in most places in `supervision`. Let's keep it this way.

Reply:
Also, I think it should be `Optional`.